src/Dofs/DofHandler.jl

```diff
@@ -392,17 +393,22 @@ function _create_sparsity_pattern(dh::DofHandler, ch#=::Union{ConstraintHandler,
 
     resize!(I, cnt)
     resize!(J, cnt)
-    V = zeros(length(I))
-    K = sparse(I, J, V)
 
-    # Add entries to K corresponding to condensation due the linear constraints
-    # Note, this requires the K matrix, which is why we can't push!() to the I,J,V
-    # triplet directly.
+    #If ConstraintHandler is given, create the condensation pattern due to affine constraints
     if ch !== nothing
         @assert isclosed(ch)
+
+        #Make V equal to ones instead of zeros because :+-operator removes entries with zeros.
+        V = ones(length(I))
+        K = sparse(I, J, V, ndofs(dh), ndofs(dh))
+        
```

Comment:
Perhaps you can pass `I`, `J`, `V` to `_condense_sparsity_pattern!` and just keep pushing to the same vectors instead?

Reply:
I can check if it faster, but _condense_sparsity_pattern! still needs the stiffness matrix so sparse(I,J,V) will still be called twice.

Reply:
Right, but it is possible the addition is slower than the second call to sparse I guess.